test/avocado/run-tests

```diff
@@ -232,12 +231,21 @@ def run_avocado(avocado_tests, verbose, browser, download_logs, sit):
         if selenium:
             selenium.kill()
 
+def _install_fmf():
+    try:
+        import fmf
+    except ImportError:
+        subprocess.check_output(["pip3", "install", "fmf"])
```

Comment:
This is something which we really don't want to do in production, it goes too much behind the user's back. We can install the python3-fmf package into our test containers. Either import it at the top, and the `ImportError` will tell local users to install it, or make it gracefully skip the fmf functionality on `except ImportError`.

Reply:
yep, this code is ugly and just there until it will be installed in test container.